### DISCUSSION TITLE
Android studio stop problem

### DISCUSSION THREAD

**Discussion by Abdelmenam-Tarek-Abdelmenam** [community/community#17382] [Copilot Conversations] [answered]
I really enjoy using it , it reduce programing time  but it sometimes stop in android studio and I must restart the ide to make it work again .

**Comment by jansorg** [comment, marked as answer]
Thanks for reporting! Could you post the logs the next time this happens?

Here's how to enable debug logging for the Copilot plugin and how to locate the log file.
1. Open `Help > Diagnostic Tools > Debug Log Settings...`
2. Add a new line `#com.github.copilot` to enable debug logging for Copilot
3. Execute the action in your IDE which isn't working as expected
4. Open the log file with `Help > Show Log in ...`
5. Review the log file and post the relevant snippets here
